### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by dmtrKovalenko**
Thanks for 3 contributors that made this release possible

Here are some highlights ✨:
* Fix missing `tslib` dependency
* Fix common typescript errors 
* Improve bundle size by removing prop types and treeshakable classes

### Fixes
* Fix missing tslib de
* Fix a bug with white border if localized dialog actions are too wide
* Fix not updating component on `initialFocusedDate` change

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -73,6 +74,7 @@
     "postinstall": "node -e \"console.log('\\u001b[35m\\u001b[1mHave you installed one of peer libraries?\\u001b[22m\\u001b[39m\\n > date-fns \\n > luxon \\n > moment')\""
   },
   "devDependencies": {
+    "@babel/core": "^7.1.2",
```
Comment (TrySound) [inline lib/package.json]: It's not used. Should be removed.

**Comment by TrySound** [comment]
All good

**Comment by dmtrKovalenko** [comment]
I will also remove `babel-core` from dependencies. And also something wrong with codecov. It says 57% on main page 😢

**Comment by TrySound** [comment]
No, keep it. I removed that comment. Just confused it with runtime.

**Pull request merged**